### PR TITLE
[12.0][ADD] mail_preview_base & mail_preview_audio

### PR DESCRIPTION
**Pull request by etobella**
@pedrobaeza as promised, I created the module we talked about on OCA/dms#15

This PR adds two modules:

`mail_preview_base` creates a unified structure in order to add previews and creates a new field widget
`mail_preview_audio` is an example of how to add a new kind of file and allows to preview audio files

In order to test the audio, it is easy, we need to go to a thread and add an audio file, now we can preview it.

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]
Tried and working properly. Not sure about the base module name, as there's no "field" involved here. Maybe it can be called `mail_preview_base`.

**Comment by etobella** [comment]
was for the widget, but I will change for base :wink:

**Comment by etobella** [comment]
Just a comment, the `mail_preview_base` must be LGPL in order to keep the compatibility to dms, as it was original LGPL

**Comment by pedrobaeza** [comment]
👍

File: mail_preview_audio/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Preview audio files',
+    'summary': """
+        Allow to preview audio files""",
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'Creu Blanca,Odoo Community Association (OCA)',
+    'website': 'www.creublanca.es',
```
Comment (jarroyomorales) [inline mail_preview_audio/__manifest__.py]: Website should be https://github.com/OCA/social/

**Review by jarroyomorales — APPROVED** [review]
Functional review and works as expected! Also did a minor comment.

**Pull request merged**